net-misc/axel/axel-2.16.1.recipe

```diff
@@ -41,10 +44,10 @@ BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:awk
 	cmd:find
-	cmd:gcc
+	cmd:gcc$secondaryArchSuffix
 	cmd:gettext
```

Comment:
```diff
-	cmd:gettext
+	cmd:gettext$secondaryArchSuffix
```
That's because we have a `$secondaryArchSuffix` for it: https://github.com/haikuports/haikuports/blob/ae4a2a91e4c906b726416de682b2c638c58a5e63/sys-devel/gettext/gettext-0.19.8.1.recipe#L42